src/legacy/core_plugins/data/public/index_patterns/index_patterns_service.ts

```diff
@@ -75,7 +75,7 @@ const ui = {
   IndexPatternSelect,
 };
 
-export { validateIndexPattern, constants, fixtures, ui };
+export { validateIndexPattern, constants, fixtures, ui, IndexPatterns };
```

Comment:
whats this for ?

Reply:
oh i see, we don't export it on the top level as its only used from another service in data plugin

Reply:
Don't you think it's still useful to export?